CFC_DataCollector/recommender/augment_trips_pipeline.py

```diff
@@ -0,0 +1,26 @@
+import get_trips
+#import Profiles
+
+def augment_trips(trip_list):
+    '''
+    store trips to database
+    '''
+    return    
+
+def get_bare_trips(user_ID, filter_queries):
```

Comment:
what are bare trips?
Also, if you assume that the filter_queries is an input, who passes it in?